Home.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+N_Second = input()
+try:
+    N_Second = int(N_Second)
+except ValueError:
+    print("Это не число")
+else:
+    second = N_Second % 60
+    N_Second = (N_Second - second) / 60
```

Comment:
Чтобы не приводить к типу int выражение в следующей строке, можно здесь использовать целочисленное деление:
```suggestion
    N_Second = (N_Second - second) // 60
```